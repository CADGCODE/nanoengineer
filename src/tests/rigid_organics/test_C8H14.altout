======= test_C8H14.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 2
== structure comparison ==
cutover to fine tolerance at 1
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_C8H14.mmp  > test_C8H14.xyz
final RMS gradient=2604.488562 after 5 iterations
== structure comparison ==
at: translate (-0.000000 -0.000000 -0.000000)
       rotate (-0.000000 -0.000000 -0.000000)
        scale 1.007494,
standard deviation = 1.630424e-04, max delta = 1.531081e-02
standard deviation exeeded deviationLimit of 1.000000e-08
maximum delta exceeded max delta limit of 1.000000e-04
scale exceeded limit of 1.000100e+00
======= test_C8H14.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_C8H14.mmp
# Output File: test_C8H14.xyz
# Trace File: test_C8H14.trc
# Number of Atoms: 22
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#
0 3593.21 4314.71
1 2667.25 2912.80
2 2604.93 2843.71
3 2604.49 2843.22
4 2604.49 2843.22
5 2604.49 2843.22
# Done: Minimization final rms: 2604.488562, highForce: 2843.217741
======= test_C8H14.xyz =======
22
RMS=2604.488562
C 0.575147 -0.231704 -1.478454
C 1.482697 -0.700663 -0.293856
C -0.754357 0.330039 -0.923071
C 0.752282 -0.456913 1.045848
C -0.458844 1.522513 0.015771
C -1.482311 -0.775275 -0.124966
C -0.573417 -1.252959 1.053443
C 0.448631 1.051040 1.195263
H 1.080411 0.544547 -2.055465
H 0.353108 -1.072319 -2.141045
H 2.426009 -0.151079 -0.289322
H 1.704899 -1.766374 -0.383357
H -1.380946 0.657280 -1.742379
H 1.381328 -0.785403 1.861717
H -1.403788 1.917170 0.391743
H 0.040805 2.311702 -0.549205
H -2.426430 -0.375760 0.249716
H -1.700255 -1.613540 -0.787461
H -0.344272 -2.317137 0.945803
H -1.080359 -1.093929 2.006236
H -0.054680 1.223659 2.148303
H 1.389345 1.606106 1.182735
