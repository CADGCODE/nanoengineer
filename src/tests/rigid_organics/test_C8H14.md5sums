stdout D0BB091C606EFD3B877E0F38497B0515
stderr D41D8CD98F00B204E9800998ECF8427E
test_C8H14.trc 242387BE8F6D5AA6EAED5D19D367FB02
test_C8H14.xyz DB0010A23E028D5ACB2B7BCB9F3F0C9D
