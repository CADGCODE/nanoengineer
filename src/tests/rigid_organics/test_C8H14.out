======= test_C8H14.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
cutover to fine tolerance at 2
== structure comparison ==
